sencha-workspace/SlateDemonstrationsTeacher/app/controller/Viewport.js

```diff
@@ -1,45 +0,0 @@
-Ext.define('SlateDemonstrationsTeacher.controller.Viewport', {
-    extend: 'Ext.app.Controller',
-
-
-    // controller configuration
-    views: [
-        'Dashboard'
-    ],
-
-    refs: {
-        dashboardCt: {
-            selector: 'slate-demonstrations-teacher-dashboard',
-            autoCreate: true,
-
-            xtype: 'slate-demonstrations-teacher-dashboard'
-        }
-    },
-
-
-    // controller templates method overrides
-    onLaunch: function () {
-        var siteEnv = window.SiteEnvironment || {},
-            contentAreaCode = (siteEnv.cblContentArea || {}).Code,
-            dashboardCt, progressGrid;
-
-        dashboardCt = this.getDashboardCt();
-        progressGrid = dashboardCt.getProgressGrid();
```

Comment:
@aliyeysides these two variable initializations must have been broken out from the declaration block in the past because there was a condition in between that could abort it. In its current form though these should be consolidated